src/hal/src/lib.rs

```diff
@@ -366,12 +369,13 @@ impl Error for SubmissionError {
 pub type SubmissionResult<T> = Result<T, SubmissionError>;
 
 
-/// Represents a handle to a physical device.
+/// Represents a handle to a physical device, such as a particular
```

Comment:
This needs to be fixed. We have a separate type `PhysicalDevice` now, and the `Gpu` struct is not that but rather a combination of the logical device and the hardware queues it provides.

Reply:
Ah, interesting, I need to look at this more.  Thank you.